build.gradle.kts

```
@@ -50,7 +67,7 @@ dependencies {
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:latest.release")
 
     testImplementation("org.openrewrite:rewrite-kotlin:${rewriteVersion}")
-    testImplementation("org.openrewrite:rewrite-maven")
+    testImplementation("org.openrewrite:rewrite-properties:${rewriteVersion}")
```

This rewrite-bom manages the version here, so we can leave that out already.
```suggestion
    testImplementation("org.openrewrite:rewrite-properties")
```